### DISCUSSION TITLE
Differenz im Mitgliedskonto mit Vorzeichen und farblich unterschiedlich

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#349] [General]
Bei uns kommt es vor, dass manchmal jemand etwas zu viel bezahlt, was dann bei der nächsten Abrechnung wieder ausgeglichen wird.
Aus dem Mitgliedskonto ist jedoch nicht ersichtlich, welches Vorzeichen die Differenz hat. Hier wäre es hilfreich, die Differenz unterscheidbar zu machen. Klar, ist nicht so einfach, da Soll - Ist meistens einen positiven Wert liefert, nur eben bei Überzahlung nicht. Vielleicht kann man die Überzahlung grün darstellen und "Schulden" rot? Dann spart man sich ein Vorzeichen und ggf. eine Fehlinterpretation.

**Comment by dippeal** [comment]
Keine Ahnung ob Farben gehen, aber man könnte bestimmt die Spalte "Differenz" in "Schulden" bzw. "Guthaben" umbenennen/hinzufügen (je nachdem).

**Comment by lenilsas** [comment]
Ich verstehe nicht genau wo du meinst.
-Unter Mitglied->Mitgliedskonto, hier wird die Differenz mit Vorzeichen angezeigt
-Unter Sollbuchungen kann nach Fehlbetrag oder Überzahlung gefiltert werden. Hier wird der Betrag und Zahlungseingang angezeigt.

**Reply by tolot27** [reply]
Das mit den Sollbuchen habe ich heute Nacht auch entdeckt. Im Mitgliedskonto gab es bei mir keine Vorzeichen, zumindest nicht, als ich einen Abrechnungslauf gemacht habe. Dann hat eh einiges nicht funktioniert und ich habe den Lauf wieder gelöscht. Später habe ich dann bei einem Mitgliedskonto tatsächlich auch eine negative Differenz gesehen, genauso wie ich es erwarten würde.
Ich prüfe das heute Abend nochmal. Mir erschien es jedenfalls ziemlich merkwürdig. Daher ja auch der Post hier.

**Reply by JohannMaierhofer** [reply]
Also bei mir ist die Differenz auch mit Vorzeichen.
